### DISCUSSION TITLE
OCR and Image Load Questions

### DISCUSSION THREAD

**Discussion by nark31** [NateScarlet/auto-derby#357] [Q&A] [answered]
It may be awkward because it is a sentence using a translator.

I'm converting it to a Korean server.
There is a phenomenon that some character recognition is grouped into two or more and printed out, perhaps due to a font problem.
![image](https://user-images.githubusercontent.com/3796369/187040695-d86bc858-d4d3-447f-aacc-b6aab20161ea.png)
Characters that should be divided into 'li', '0', '0', '1', and '2' are combined and recognized like a picture.
Is there a solution to the above problem?
Also, I want to change the image to match the Korean server, so is there a function to change the file in real time and load it?
Sometimes the recognition rate is low even though it is the same image/coordinate.

Thank you.

**Comment by NateScarlet** [comment, marked as answer]
text segementation is [hand written](https://github.com/NateScarlet/auto-derby/blob/c18c8f37dfa315fe903c5efc362a4a5cfc3bcb4a/auto_derby/ocr.py#L200-L307) by my self

enable verbose log by set env var `DEBUG` to `auto_derby.ocr` 

you can try do the same or use a third party ocr engine

**Reply by NateScarlet** [reply]
i don't undersand second question well.

if you want reload template without restart, call `auto_derby.template._LOADED_TEMPLATES.clear()`
